libafl/src/executors/forkserver.rs

```diff
@@ -421,6 +438,11 @@ impl Forkserver {
             command.stderr(Stdio::null());
         }
 
+        if let Some(core) = core {
+            command.bind(core);
+        }
+
+        command.env("AFL_MAP_SIZE", format!("{coverage_map_size}"));
```

Comment:
AFL_MAP_SIZE_ENV_VAR = "AFL_MAP_SIZE" so you don't need this line

Reply:
Thanks for pointing this out. I think I accidentally add this line and will remove.

Reply:
Oh this goes here when I resolved the conflicts =p